build/ci/ego

```diff
@@ -268,7 +261,7 @@ resolve_system_dependencies() {
     if [[ "${PLATFORM}" = "debian" ]] || [[ "${PLATFORM}" = "ubuntu" ]]; then # Debian/Ubuntu
         _wait_for_process_to_finish apt
         _wait_for_process_to_finish dpkg
-        apt-get install -y -q --no-install-recommends ca-certificates curl logrotate openssl snmp
```

Comment:
`ca-certificates` was giving some issues, also curl and logrotate are already installed so I'm cleaning this up